superset-frontend/src/components/Pagination/PaginationWrapper.test.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { supersetTheme, ThemeProvider } from '@superset-ui/core';
+import { PaginationWrapper } from './PaginationWrapper';
+
+test('PaginationWrapper rendering correctly', () => {
+  render(
+    <ThemeProvider theme={supersetTheme}>
```

Comment:
`ThemProvider` and `import '@testing-library/jest-dom'` would not be needed if you just `import { render } from 'spec/helpers/testing-library'`.

Reply:
They are necessary...

No ThemeProvider:
<img width="974" alt="Captura de Tela 2021-02-23 às 11 01 45 AM" src="https://user-images.githubusercontent.com/1014611/108854538-c4c2b080-75c6-11eb-929c-011863149eda.png">

No `@testing-library/jest-dom`:
<img width="668" alt="Captura de Tela 2021-02-23 às 11 02 31 AM" src="https://user-images.githubusercontent.com/1014611/108854550-c8eece00-75c6-11eb-9bf0-b6b0daeab086.png">

Reply:
Very good to know :-)